README.md

```diff
@@ -90,44 +106,32 @@ Below are the features we have implemented to date.
     </picture>
 </p>
 
-# Upcoming features
+<br>
+
+# What's In Store
+
+Here’s what you can look forward to:
 
-Coming next:  
-⚙️  A notification center<br>
-⚙️  More standard fields<br>
-⚙️  AI/LLM integration to Notes<br>
-⚙️  More pipelines and table customization<br>
-⚙️  A lot of bug fixes and other improvements<br>
-⚙️  Auto-logging emails<br>
+⏳ **Frequent updates:** We’re shipping fast! Expect regular updates and new features that enhance your CRM experience.
 
-Coming later this year:  
-📅  Advanced search and filters<br>
-📅  Data connectors<br>
-📅  Custom fields<br>
-📅  Custom objects<br>
-📅  +++<br>
+🔗 **Extensibility:** We’re putting the power in your hands. Soon, you’ll have the tools to extend and customize your CRM to precisely fit your business needs. No more reliance on external consultants; you’re in control.
 
-Track planning on [Github Projects](https://github.com/orgs/twentyhq/projects/1) and help us prioritising by upvoting or creating [issues](https://github.com/twentyhq/twenty/issues)
+<br>
+
+# Join the Movement
 
-# Documentation
-The doc is available on [docs.twenty.com](https://docs.twenty.com).
+- Star the repo
+- Join [discussions](https://github.com/twentyhq/twenty/discussions) and track [issues](https://github.com/twentyhq/twenty/issues) 
+- Follow us on [Twitter](https://twitter.com/twentycrm) or [LinkedIn](https://www.linkedin.com/company/twenty/) 
+- Join our [Discord](https://discord.gg/cx5n4Jzs57)
+- [Contributions](https://github.com/twentyhq/twenty/contribute) are, of course, most welcome! 
 
-If you have any question feel free to ask on our [Discord](https://discord.gg/cx5n4Jzs57).
+<br>
 
-# Helping us
-- The easiest way to help us is to star the repo and helping us getting visibility.
-- Contributions are, of course, most welcome!  
-It doesn't have to be code, you could create an [issue](https://github.com/twentyhq/twenty/issues) to report a bug you've spotted, join [discussions](https://github.com/twentyhq/twenty/discussions), [Discord](https://discord.gg/cx5n4Jzs57) or write [documentation](https://docs.twenty.com/).
+# Get Started
 
+Dive into Twenty today and experience the power of open-source CRM on your own terms.
 
-# Office hours
-Want to meet the team? Get direct support?
-Feel free to book a slot during our office hours.
+🚀 Get Started with [Twenty](https://docs.twenty.com/developer/local-setup). 
 
-<a href="https://cal.com/team/twenty?utm_source=banner&utm_campaign=oss">
-  <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://cal.com/book-with-cal-dark.svg">
-    <source media="(prefers-color-scheme: light)" srcset="https://cal.com/book-with-cal-light.svg">
-    <img alt="Book us with Cal.com" src="https://cal.com/book-with-cal-light.svg" height="40">
-  </picture>
-</a>
+Welcome to the future of CRM. Welcome to Twenty.
```

Comment:
Maybe I'd get rid of this as it's not necessary and a bit too much marketing speak